src/main/java/Parser.java

```diff
@@ -0,0 +1,83 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+
+    /**
+     * Parses the command input by the user and returns the action to be carried out.
+     *
+     * @param command Command input by the user.
+     * @return Action to be carried out.
+     */
+    public static String parseCommand(String command) {
+        return command.split(" ")[0];
+    }
+
+    /**
+     * Returns the index of the task to be marked as done.
+     *
+     * @param command Command input by the user.
+     * @return Index of task to be marked as done.
+     */
+    public static int parseDoneIndex(String command) {
+        return Integer.parseInt(command.split(" ")[1]) - 1;
+    }
+
+    private static LocalDateTime DateTimeParser(String input) {
+        return LocalDateTime.parse(input,
+                DateTimeFormatter.ofPattern("dd-MM-yyyy HHmm"));
+    }
+
+    /**
+     * Parses the stored string and finds the type of task.
+     *
+     * @param task String retrieved from Storage.
+     * @return Type of task.
+     */
+    public static String parseTypeOfTask(String task) {
+        String[] split = task.split("\\|");
+        String typeOfTask = split[0].trim();
+        return typeOfTask;
+    }
+
+    /**
+     * Parses the stored string and finds the completion status.
+     *
+     * @param task String retrieved from Storage.
+     * @return Completion status of the task.
+     */
+    public static String parseCompletionStatus(String task) {
+        String[] split = task.split("\\|");
+        String isDone = split[1].trim();
+        return isDone;
+    }
```

Comment:
The variable isDone sounds like a boolean return type instead of a String, perhaps it will be better to rename the variable.